lib/core/network/api/services/remote_config_service.dart

```diff
@@ -2,9 +2,25 @@ import 'dart:convert';
 
 import 'package:firebase_remote_config/firebase_remote_config.dart';
 
-enum Networks { telos, telosTestnet, eos, eosTestnet }
+enum Network {
```

Comment:
Changed name and added their string value.

Reply:
@n13  pls check the enum maps to the correct string value. At the end, when the calls are made, the string is what is being passed to the call

Reply:
Don't need to do that see below